src/App.tsx

```diff
@@ -1,61 +1,131 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+const allowedCharacters
+  = 'abcdefghijkmnopqrstuvwxyzабвгґдеєжзиіїйклмнопрстуфхцчшщьюя123456789 ';
+
+function isCharacterAllowed(char: string) {
+  if (allowedCharacters.includes(char.toLocaleLowerCase())) {
+    return true;
+  }
+
+  return false;
+}
 
 export const App = () => {
+  const [toDos, addToDo] = useState([...todosFromServer]);
```

Comment:
```suggestion
  const [todos, setTodos] = useState([...todosFromServer]);
```
Same below. Function name must be set according to the noun variable.
Please fix all cases